apps/website/app/(docs)/docs/obsidian/docMap.ts

```diff
@@ -0,0 +1,17 @@
+const OBSIDIAN_DOCS = "app/(docs)/docs/obsidian/pages";
+const SHARED_DOCS = "app/(docs)/docs/sharedPages";
+
+type DocMapType = {
+  default: string;
+  [key: string]: string;
+};
+
+export const docMap: DocMapType = {
```

Comment:
Can we generalize this so that we don't need to duplicate it for Roam?
If we add another shared doc, right now we'd need to add it in two places.